tests/h/schemas/admin_organization_test.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import unicode_literals
+
+import colander
+import pytest
+
+from h.schemas.admin_organization import OrganizationSchema
+
+
+class TestOrganizationSchema(object):
+    def test_it_allows_valid_data(self, org_data, bound_schema):
```

Comment:
Is there any value for testing for name validation (too short, too long?)?

Reply:
The minimum length is one char, so it can't be too short. It can be too long however. I'll add a test for this.